.github/workflows/release_wheel_aarch64.yml

```diff
@@ -35,7 +35,11 @@ jobs:
           - cuda: "12.8"
             torch: "2.7"
             python: "3.12"
-            image: "nvcr.io/nvidia/pytorch:25.03-py3"
+            image: "nvcr.io/nvidia/pytorch:25.02-py3"
+          - cuda: "12.9"
```

Comment:
PyTorch do not have a native build of 12.9 so for x86_64 we don't need a cu129 wheel, for arm64 we follow the nvcr container so it make sense to have a cu129 wheel (at least for now).